examples/opensora_hpcai/opensora/pipelines/infer_pipeline.py

```diff
@@ -58,10 +58,10 @@ def __init__(
         else:
             self.sampling_func = self.diffusion.p_sample_loop
 
-    @ms.jit
+    # @ms.jit
     def vae_encode(self, x: Tensor) -> Tensor:
-        image_latents = self.vae.encode(x)
-        image_latents = image_latents * self.scale_factor
+        # image_latents = ops.stop_gradient(self.vae.encode(x))
+        image_latents = ops.stop_gradient(self.vae.module.encode(x) * self.vae.scale_factor)
```

Comment:
the scale factor is independent of type of the vae to use, so I think better to keep the parameter in difussion pipeline scope instead of vae scope

Reply:
once it is parsed to vae, it is as a member. it should be configurable but hpcai write it in the code and train it with this fixed value... which is not so proper for different vae training data.

Reply:
I get this: `AttributeError: The 'VideoAutoencoderPipeline' object has no attribute 'module'.`

Reply:
And this: `AttributeError: The 'VideoAutoencoderPipeline' object has no attribute 'scale_factor'.`